protos/flyteidl/core/types.proto

```diff
@@ -67,6 +67,11 @@ message StructuredDatasetType {
     // The serialized bytes of a third-party schema library like Arrow.
     // This is an optional field that will not be used for type checking.
     bytes external_schema_bytes = 4;
+    
+    // This indicates which columns in the structured dataset is used for
+    // partitioning when the dataset is serialized. This is an optional field
+    // that will not be userd for type checking
```

Comment:
datset is used -> dataset *are* used

userd -> used

period on the end?

Reply:
Nice catch! I think dataset is singular :)

Reply:
but columns is plural right?

Reply:
🤦‍♂️